corehq/apps/accounting/tests/test_customer_invoicing.py

```diff
@@ -511,98 +441,66 @@ def test_under_limit(self):
+        num_sms_non_main_domain1 = random.randint(self.sms_rate.monthly_limit + 1,
+                                          self.sms_rate.monthly_limit + 2)
+        non_main_domain1_billables = arbitrary_sms_billables_for_domain(
+            self.non_main_domain1, self.sms_date, num_sms_non_main_domain1
         )
 
         sms_line_items = self._create_sms_line_items()
-        self.assertEqual(sms_line_items.count(), 2)
+        self.assertEqual(sms_line_items.count(), 1)
         for sms_line_item in sms_line_items:
             self.assertIsNone(sms_line_item.base_description)
             self.assertEqual(sms_line_item.base_cost, Decimal('0.0000'))
             self.assertEqual(sms_line_item.quantity, 1)
 
-            if self.advanced_rate.feature == sms_line_item.feature_rate.feature:
-                sms_cost = sum(
-                    billable.gateway_charge + billable.usage_charge
-                    for billable in advanced_billables[self.advanced_rate.monthly_limit:]
-                )
-            else:
-                sms_cost = sum(
-                    billable.gateway_charge + billable.usage_charge
-                    for billable in billables[self.sms_rate.monthly_limit:]
-                )
+            sms_cost = sum(
+                billable.gateway_charge + billable.usage_charge
+                for billable in (
+                    non_main_domain1_billables[self.sms_rate.monthly_limit:]
+                    + main_domain_billables
+                ))
             self.assertEqual(sms_line_item.unit_cost, sms_cost)
             self.assertEqual(sms_line_item.total, sms_cost)
 
     def test_subscription_level_sms_credits(self):
         # Add SMS usage
         arbitrary_sms_billables_for_domain(
-            self.domain, self.sms_date, self.sms_rate.monthly_limit + 1
+            self.main_domain, self.sms_date, self.sms_rate.monthly_limit + 1
         )
         arbitrary_sms_billables_for_domain(
-            self.domain2, self.sms_date, num_sms=self.advanced_rate.monthly_limit + 10
-        )
-
-        # Cover the cost of 1 SMS on the Standard subscription
-        CreditLine.add_credit(
-            amount=Decimal(0.7500),
-            feature_type=FeatureType.SMS,
-            subscription=self.subscription
-        )
-        # Cover the cost of 10 SMS on the Advanced subscription
-        CreditLine.add_credit(
-            amount=Decimal(7.5000),
-            feature_type=FeatureType.SMS,
-            subscription=self.sub2,
+            self.non_main_domain1, self.sms_date, num_sms=10
         )
 
-        calculate_users_in_all_domains(self.invoice_date)
-        tasks.generate_invoices_based_on_date(self.invoice_date)
-        self.assertEqual(CustomerInvoice.objects.count(), 1)
-        invoice = CustomerInvoice.objects.first()
-        self.assertEqual(invoice.balance, Decimal('1500.0000'))
-
-    def test_one_subscription_level_sms_credit(self):
```

Comment:
Same

Reply:
Add them back. Replied in this [comment.](https://github.com/dimagi/commcare-hq/pull/33627/files/5106ad3deb88a1fe4c668bd9c1939b43d00dbf32#r1367827517)